app/templates/admin/school-counts.jade

```diff
@@ -10,7 +10,7 @@ block content
     h3 Loading...
     p Change URL query string ?batchsize=#{view.batchSize} above to be smaller if this page never loads or larger to speed it up (currently set to #{view.batchSize})
     p
-      !{view.state}
+    | !{view.state}
```

Comment:
Preserves behavior and solves an error that appears in the webpack build.
Reference: https://stackoverflow.com/a/29939097